src/middlewares/worker.js

```diff
@@ -1,7 +1,9 @@
 import createWorkerMiddleware from 'redux-worker-middleware';
 
-/* eslint-disable import/no-webpack-loader-syntax */
+/* eslint-disable */
```

Comment:
Would it not be better to only disable `import/no-webpack-loader-syntax`, rather than all of eslint?